frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/services/ReverificationResultService.java

```diff
@@ -83,41 +84,50 @@ public TokenRequest constructTokenRequest(String authCode) {
     }
 
     public TokenResponse sendTokenRequest(TokenRequest tokenRequest) {
-        try {
-            LOG.info("Sending IPV token request");
-            int count = 0;
-            int maxTries = 2;
-            TokenResponse tokenResponse;
-
-            do {
-                if (count > 0) {
-                    LOG.warn("Retrying IPV access token request");
-                }
-
-                count++;
+        LOG.info("Sending IPV token request");
+        int count = 0;
+        TokenResponse tokenResponse =
+                new TokenResponse() {
+                    @Override
+                    public boolean indicatesSuccess() {
+                        return false;
+                    }
+
+                    @Override
+                    public HTTPResponse toHTTPResponse() {
+                        return null;
+                    }
```

Comment:
Yes, need to provide implementations for these methods in order to create an instance.  This is a temporary change that is the first step to stop leaking implementation details from this class.  Next change will be to return a ResultOrError construct rather than a TokenRequest.